projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.component.html

```diff
@@ -0,0 +1,84 @@
+<ng-container *ngIf="messageEvents$ | async as messageEvents">
+  <div class="container">
+    <div class="cx-avatar-line"></div>
+    <div class="cx-messages" id="cx-messages">
+      <ng-container *ngFor="let message of messageEvents">
+        <div
+          class="cx-message-card"
+          [ngClass]="{ 'cx-message-display': message.rightAlign }"
+        >
+          <cx-avatar [message]="message"> </cx-avatar>
+
+          <div>
+            <label>{{
+              message.createdAt | cxDate: 'MMMM d, YYYY h:mm aa'
```

Comment:
Does this format need to be used on all customer ticketing pages?
If so, then can we extract to a central customer ticketing formatter?
Even mobile view needs the same format
@adolfobee @manpreetlamba4

Reply:
Yes, it will be used at 2 places in customer ticketing. we can use a constant for that.
I am adding that into the PR for abstract dialog component.

Reply:
we can declare the same constant here as well.

Reply:
Also, adding dateFormat to config so it can be passed from parent